src/components/atoms/SessionLabel/index.tsx

```diff
@@ -0,0 +1,39 @@
+import { Box, Typography } from '@mui/material'
+import { Colors } from 'src/styles/color'
+
+type Props = {
+  sessionId: string
+  room: 'a' | 'b'
+  isRoomNameDisplayed: boolean
+}
+
+const getLabelStyle = (room: Props['room']) => {
+  switch (room) {
+    case 'a':
+      return { backgroundColor: Colors.background.primary_pink, roomName: 'Room A' } as const
+    case 'b':
+      return { backgroundColor: Colors.background.primary_green, roomName: 'Room B' } as const
+    default:
+      throw new Error(`Invalid room ${room}`)
+  }
+}
```

Comment:
SessionCard の方でコメントした通り、room prop を `roomName: 'Room A' | 'Room B'` とするのであればこの関数で roomName は返さなくて良くなります。

```suggestion
const getLabelStyle = (roomName: Props['roomName']): { backgroundColor: string } => {
  switch (roomName) {
    case 'a':
      return { backgroundColor: Colors.background.primary_pink } as const
    case 'b':
      return { backgroundColor: Colors.background.primary_green } as const
    default:
      throw new Error(`Invalid room name: ${roomName}`)
  }
}
```